packages/terra-worklist-data-grid/src/proptypes/cellShape.js

```diff
@@ -4,7 +4,8 @@ const cellShape = PropTypes.shape({
   /**
    * String indicating content to be displayed for the cell.  Content will be ignored if isMasked property is true.
    */
-  content: PropTypes.string,
+  content: PropTypes.node,
```

Comment:
Is there a reason we are using a `content` prop instead of `children`? It seems more intuitive now that `Cell` is its own subcomponent to wrap the content like this:

```js
<Cell>{someContent}</Cell>
```

Reply:
You have an array of cells being passed as a structure.  Consumers would not be creating our internal Cell React component

Reply:
This would be done internally. Example (from [Row.jsx](https://github.com/cerner/terra-framework/pull/1676/files#diff-542c04ec82fe30d4fd6235a7ac912256c6469f2f05a311238bc20ee1bd5ac9e4R94-R105)):

```js
      <Cell
        rowId={rowId}
        columnId={columnId}
        rowIndex={cellRowIndex}
        columnIndex={cellColumnIndex}
        key={`${rowId}_${columnId}`}
        isTabStop={tabStopColumnIndex === cellColumnIndex}
        isSelected={!hasSelectableRows && selectedCellColumnId === columnId}
        cell={cell}
        isRowHeader={isRowHeader}
        onCellSelect={handleCellSelect}
      >
        {someContent}
      </Cell>
```

Reply:
+1 to Sydney's suggestion as it's consistent with Terra conventions.